androidlibrary_lib/src/androidTest/java/org/opendatakit/database/data/TableMetaDataEntriesTest.java

```diff
@@ -0,0 +1,105 @@
+package org.opendatakit.database.data;
+
+import android.os.Parcel;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+@RunWith(AndroidJUnit4.class)
+public class TableMetaDataEntriesTest {
+
+    //
+    TableMetaDataEntries testMetaDataEntries;
+    private final String TABLE_ID = "testTable";
+    private final String REV_ID = "revId";
+    KeyValueStoreEntry entry1 = new KeyValueStoreEntry();
+    KeyValueStoreEntry entry2 = new KeyValueStoreEntry();
+
+    @Before
+    public void setUp() {
+        testMetaDataEntries = new TableMetaDataEntries(TABLE_ID, REV_ID);
+
+        // first test entry
+
+        entry1.tableId = TABLE_ID;
+        entry1.partition = "partition1";
+        entry1.aspect = "aspect1";
+        entry1.key = "key1";
+        entry1.type = "number";
+        entry1.value = "value1";
```

Comment:
use string constants so you can then reuse string constants in assert statements